website/docs/r/oracle_exascale_database_virtual_machine_cluster.html.markdown

```diff
@@ -0,0 +1,205 @@
+
+* `single_client_access_name_listener_port_tcp_ssl` - (Optional) The TCPS Single Client Access Name (SCAN) port. Defaults to `2484`. Changing this forces a new Exadata VM Cluster to be created.
+
+* `system_version` - (Optional) Operating system version of the Exadata image. System version must be <= Db server major version (the first two parts of the DB server version eg 23.1.X.X.XXXX). Changing this forces a new resource to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Exadata VM Cluster.
+
+* `time_zone` - (Optional) The time zone of the Exadata VM Cluster. For details, see [Exadata Infrastructure Time Zones](https://docs.cloud.oracle.com/iaas/Content/Database/References/timezones.htm). Changing this forces a new Exadata VM Cluster to be created.
+
+---
+
+A `virtual_machine_file_system_storage` block supports the following:
+
+* `total_size_in_gb` - (Required) Total Capacity. Changing this forces a new resource to be created.
+
+---
+
+A `data_collection` block supports the following:
+
+* `diagnostics_events_enabled` - (Optional) Indicates whether diagnostic collection is enabled for the VM cluster, Cloud VM cluster or VMBM DBCS. Changing this forces a new Cloud VM Cluster to be created.
+
+* `health_monitoring_enabled` - (Optional) Indicates whether health monitoring is enabled for the VM cluster, Cloud VM cluster or VMBM DBCS. Changing this forces a new Cloud VM Cluster to be created.
+
+* `incident_logs_enabled` - (Optional) Indicates whether incident logs and trace collection are enabled for the VM cluster, Cloud VM cluster or VMBM DBCS. Changing this forces a new Cloud VM Cluster to be created.
+
+---
+
+A `network_security_group_cidr` block exports the following:
+
+* `destination_port_range` - (Required) A `destination_port_range` block as defined below.
+
+* `source` - (Required) It is a range of IP addresses that a packet coming into the instance can come from. Changing this forces a new resource to be created.
+
+---
+
+A `destination_port_range` block exports the following:
+
+* `max` - (Required) The maximum port number, which must not be less than the minimum port number. Changing this forces a new resource to be created.
+
+* `min` - (Required) The minimum port number, which must not be greater than the maximum port number. Changing this forces a new resource to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Exadata VM Cluster.
+
+* `ocid` - The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the Exadata VM Cluster.
+
+* `zone_ocid` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the zone the Exadata VM Cluster is associated with.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/language/resources/syntax#operation-timeouts) for certain actions:
```

Comment:
This needs to be updated to align with the latest [guidelines](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-new-resource.md#step-8-add-documentation-for-this-resource)
```suggestion
The `timeouts` block allows you to specify [timeouts](https://developer.hashicorp.com/terraform/language/resources/configure#define-operation-timeouts) for certain actions:
```